### ISSUE TITLE
Taylor diagram plot

### ISSUE THREAD

**Issue by rkouznetsov** [metno/pyaerocom#1130]
TL;DR: Taylor diagram at https://aeroval.met.no/pages/overall/?project=cams2-82&experiment=IFS&tab=taylor looks like:
![aod-all-2013-2024](https://github.com/metno/pyaerocom/assets/11734464/a97d371a-5f24-4e02-b437-8931b1ffc8f0)
It should be like one at:
https://en.wikipedia.org/wiki/Taylor_diagram#/media/File:Primer_fig1a.svg

Description:
The Taylor's paper describes a diagram designed so that de-biased RMSE is equals
to the distance from Ref to a point, and the Correlation axis (arc) has
arcsin scaling. In your plots the arc is scaled linearly, so that the distance
from the ref means something else. Along with fixing the arc
scaling, it would be great also to put isolines of the distance, which should
have a shape of circular arcs, and put a reference to the paper,
similar to what you have done for the FAIRMODE plots at https://aeroval.met.no/pages/overall/?project=cams2-83&experiment=forecast-last-seasons&tab=target.

I hope it is not too difficult to fix this issue.

Thank you for the great evaluation tool!

**Comment by jgriesfeller** [comment]
Thanks @rkouznetsov for submitting this issue and spotting the error. I guess we have not used the plot very much in our evaluations. We'll discuss internally and let you know when there's an update.
Adding a proper picture of how it is supposed to look to ease discussion:
![image](https://github.com/metno/pyaerocom/assets/10772065/21574c6b-38ad-49de-8d4e-d95839874800)

**Comment by michaelgau** [comment]
It's on the AeroVal side. I'm in contact with Augustin Mortier. There's no quick fix, but we'll get there, before the CAMS GA for sure. Thanks again @rkouznetsov for pointing this out.

**Comment by michaelgau** [comment]
Augustin has fixed the arcsin scale, so now the distance from REF should be equal to the centered RMSE :)

**Comment by michaelgau** [comment]
possibilities to draw circles around REF are being looked at..

**Comment by rkouznetsov** [comment]
Thank you! Looks way better now.  Isolines of centered RMSE would make it perfect.  Also the description seems misleading: correlation is not the angle, but rather cosine of it (or sine if the angle is counted as in meteorology).

**Comment by michaelgau** [comment]
maybe we can say that the angle _indicates_ the correlation (I didn't write that sentence, but I agree that it shouldn't be too long either). Reference to Taylor is being added.

**Comment by rkouznetsov** [comment]
Makes sense. Thank you!
